bundles/framework/myplaces3/view/MainView.js

```diff
@@ -123,6 +123,7 @@ Oskari.clazz.define("Oskari.mapframework.bundle.myplaces3.view.MainView",
                             this._setMeasurementResult(event.getData());
                         }
                     }else if(this.instance.isFinishedDrawing()){
+                        console.log(event);
```

Comment:
Remove console.log. You can use Oskari.log('loggername').debug(msg) instead if you add a little context to the message.